### PR TITLE
refactor(cdnjs): Use single cache namespace

### PR DESCRIPTION
**Pull request by zharinov**
## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/datasource/cdnjs/index.ts [hunk 1]
```diff
@@ -34,7 +34,10 @@ export class CdnjsDatasource extends Datasource {
 
   @cache({
     namespace: `datasource-${CdnjsDatasource.id}`,
-    key: ({ packageName }: GetReleasesConfig) => packageName.split('/')[0],
+    key: ({ packageName }: GetReleasesConfig) => {
+      const library = packageName.split('/')[0];
```
Comment (viceice) [inline lib/modules/datasource/cdnjs/index.ts:38]: do we really need this extra variable?

Comment (zharinov) [reply lib/modules/datasource/cdnjs/index.ts:38]: It's not instantly obvious what does it mean, and looks ugly in the string template literal

Comment (viceice) [reply lib/modules/datasource/cdnjs/index.ts:38]: we should later extract it to a static function

**Review by viceice — APPROVED** [review]

File: lib/modules/datasource/cdnjs/index.ts (hunk 1 above)
Comment (zharinov) [reply lib/modules/datasource/cdnjs/index.ts:38]: Static functions won't be available in the stage 3 decorators

**Pull request merged**